dace/frontend/python/newast.py

```diff
@@ -2509,14 +2508,17 @@ def visit_While(self, node: ast.While):
             self._generate_orelse(loop_region, postloop_block)
 
         self.last_block = loop_region
+        self._connect_break_blocks(loop_region)
+
+    def _connect_break_blocks(self, loop_region: LoopRegion):
+        for node, parent in loop_region.all_nodes_recursive(lambda n, _: not isinstance(n, (LoopRegion, SDFGState))):
+            if isinstance(node, BreakBlock):
+                for in_edge in parent.in_edges(node):
+                    in_edge.data.assignments["did_break_" + loop_region.label] = "1"
```

Comment:
I wonder if we should add a `__dace_` prefix to avoid name clashes. This is also true in the _generate_orelse code.

Reply:
Good point, agree - done.